NIPTool/build/document.py

```diff
@@ -0,0 +1,34 @@
+from NIPTool.models.constants import SAMPLE_KEYS, BATCH_KEYS
+from NIPTool.models.converters import convert
+
+
+def build_document(csv_data: dict, document_keys: list) -> dict:
+    """Build a general document based on convert models"""
+
+    document = {}
+    for key in document_keys:
+        value = csv_data.get(key)
+        value = convert(key, value)
+        if value is None:
+            continue
+        document[key] = value
+
+    return document
+
+
+def build_sample(sample_data: dict) -> dict:
+    """Builds a document for the sample collection"""
+
+    sample = build_document(sample_data, SAMPLE_KEYS)
+    sample["_id"] = sample_data.get("SampleID")
```

Comment:
Is it ok for this to be `None`? If not you might want to check if `"SampleID"` is in `sample_data` first.

Reply:
It is being handeld earlier in the code in [parse](https://github.com/Clinical-Genomics/NIPTool/blob/tests/NIPTool/parse/batch.py) through the validation_schema
So SampleID can not be None when the build code is running.

Reply:
Ok good! But why are you then using `get`? That is what confuses me, get would be used when we are uncertain if the key exists. So please change to 
```suggestion
    sample["_id"] = sample_data["SampleID"]
```